Cubical/Algebra/Polynomials/UnivariateList/Properties.agda

```diff
@@ -639,6 +639,38 @@ module PolyModTheory (R' : CommRing ℓ) where
   prod-Xn-prod (suc n) m x y = cong₂ _Poly+_ ((0rLeftAnnihilatesPoly (prod-Xn m y)) ∙ drop0) (cong (λ X → 0r ∷ X) (prod-Xn-prod n m x y))
 
 
+---------------------------------------------------------------------------
+-- The indeterminate is a regular element, i.e. multiplication with it is injective
+
+  prod-X-injective : (p : Poly R') → prod-Xn 1 p ≡ 0P → p ≡ 0P
+  prod-X-injective p Xp≡0 = p≡0
+    where
+      shift = shiftFun (Poly→PolyFun p)
+
+      0r∷p≡shift : Poly→PolyFun (0r ∷ p) ≡ shift
```

Comment:
If `shiftFun` is exported, then perhaps this lemma (read as a property of `shiftFun`) should also be exported. Do you agree?

Reply:
Yes, I agree...

Reply:
Done